compliance/locker.py

```diff
@@ -180,11 +183,10 @@ def init(self):
 
     def logger_init_msgs(self):
         """Log locker initialization information."""
-        self.logger.info(f'Local locker location is {self.local_path}')
         gpgsign = self.gitconfig.get('commit', {}).get('gpgsign')
         if self._do_push and not gpgsign:
             self.logger.warning(
-                'Commits will not be cryptographically signed, best '
+                'Commits may not be cryptographically signed, best '
```

Comment:
Under what circumstances would the be signed if gpgsign is not set?

Reply:
When running locally you're in control of your commit signing regardless of whether you've added it to the framework config.  Adding it to the framework config ensure that the commits are signed when executed from a CI.  So the warning message is misleading if you ever run from your own environment.  That's why I figured we switch "will" to "may" as the message is misleading.  It isn't checking whether your commits will be "cryptographically signed" it is just checking whewther you've included `gpgsign` in the framework execution json.

Reply:
Ah, gotcha